src/hotspot/share/opto/vectorIntrinsics.cpp

```diff
@@ -369,7 +369,8 @@ bool LibraryCallKit::inline_vector_shuffle_iota() {
     res = gvn().transform(VectorNode::make(Op_MulI, res, bcast_step, num_elem, elem_bt));
   } else if (step_val->get_con() > 1) {
     Node* cnt = gvn().makecon(TypeInt::make(log2i_exact(step_val->get_con())));
-    res = gvn().transform(VectorNode::make(Op_LShiftVB, res, cnt, vt));
+    Node* shift_cnt = gvn().transform(new LShiftCntVNode(cnt, vt));
```

Comment:
Use "vector_shift_count"?  It will mask the shift count before generating the `LShiftCntVNode`.

Reply:
`vector_shift_count` which calls `VectorNode::shift_count` does not contain `Op_LShiftB`:

```c++
VectorNode* VectorNode::shift_count(int opc, Node* cnt, uint vlen, BasicType bt) {
  // Match shift count type with shift vector type.
  const TypeVect* vt = TypeVect::make(bt, vlen);
  switch (opc) {
  case Op_LShiftI:
  case Op_LShiftL:
    return new LShiftCntVNode(cnt, vt);
  case Op_RShiftI:
  case Op_RShiftL:
  case Op_URShiftB:
  case Op_URShiftS:
  case Op_URShiftI:
  case Op_URShiftL:
    return new RShiftCntVNode(cnt, vt);
  default:
    fatal("Missed vector creation for '%s'", NodeClassNames[opc]);
    return NULL;
  }
}
```

Reply:
Thank you for your comment. I think it‘s a bit strange here to use `LShfitI` instead of `LShiftB`. Anyway, reusing is better than new things.